src/web/components/Input/StyledCheckbox.tsx

```diff
@@ -1,9 +1,20 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import * as Checkbox from '@radix-ui/react-checkbox';
-import { FieldPath, FieldValues, useController, UseControllerProps } from 'react-hook-form';
+import {
+  FieldPath,
+  FieldValues,
+  useController,
+  UseControllerProps,
+  UseFormRegisterReturn,
+} from 'react-hook-form';
 
 import './MultiCheckboxInput.scss';
 
+export function withoutRef<TFieldName extends string>(props: UseFormRegisterReturn<TFieldName>) {
```

Comment:
I'm not sure if we even still need this - it was to solve a TypeScript problem but perhaps the problem doesn't exist any more.

Reply:
If we do still need it, it probably belongs somewhere shared, rather than with a specific input component.

Reply:
withoutRef removed everywhere